facturacion_mexico/cfdi_recibidos/api.py

```
@@ -0,0 +1,58 @@
+
+	results = []
+
+	# Soporta campo "files" (múltiples) o "file" (singular)
+	file_list = files.getlist("files") or files.getlist("file") or list(files.values())
+
+	for uploaded_file in file_list:
+		file_name = getattr(uploaded_file, "filename", "cfdi.xml") or "cfdi.xml"
+		try:
+			xml_bytes = uploaded_file.read()
+			result = ingest_xml(xml_bytes, company, file_name=file_name)
+		except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)